File: Backend/TiendaLibreAPI_DAL/Models/Product.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TiendaLibreAPI_DAL.Models
{
    public class Product
    {
        public long ProductId { get; set; }
        public string ProductName { get; set; }
        public string ProductCategory { get; set; }
        public string ProductBrand { get; set; }
        public string ProductModel { get; set; }
        public float Price { get; set; }
        public int StockQty { get; set; } = 1;

        public User ProductUserSeller { get; set; }
    }
}

File: Frontend/TiendaLibre/src/app/_interfaces/product/product.ts
export interface ProductImage{
    productImageId: number,
    productImageName: string,
    dataImage: string
}

export interface ProductInfo{
    productId: number,
    productName: string,
    productCategory: string,
    productBrand: string,
    productModel: string,
    price: number,
    stockQty: number,
    userSellerId: number,
    productImages: ProductImage[]
}

